src/objectTranslations/Account-de.objectTranslation

```diff
@@ -57,58 +57,58 @@
         <name>Grantmaker__c</name>
     </fields>
     <fields>
-        <help>Der Name des Verwalters von zugewiesenen Spenden des Unternehmens</help>
-        <label>Name d. Verwalters von zugew. Spenden</label>
+        <help>Der Name des Verwalters von übereinstimmenden Schenkungen des Unternehmens</help>
+        <label>Name d. Verwalters v. übereinst. Schenkungen</label>
         <name>Matching_Gift_Administrator_Name__c</name>
     </fields>
     <fields>
-        <help>Der maximale Betrag, den das Unternehmen jedem einzelnen Geschenk zuweist.</help>
-        <label>Zugewiesener Spendenbetrag maximal</label>
+        <help>Der maximale Betrag, den das Unternehmen mit jeder Schenkung abgleicht.</help>
+        <label>Übereinstimmender Schenkungsbetrag maximal</label>
         <name>Matching_Gift_Amount_Max__c</name>
     </fields>
     <fields>
-        <help>Der kleinste Betrag, den das Unternehmen in einer einzelnen Spende zuweist.</help>
-        <label>Zugewiesener Spendenbetrag minimal</label>
+        <help>Der kleinste Betrag, den das Unternehmen mit einer einzelnen Spende abgleicht.</help>
+        <label>Übereinstimmender Mindest-Schenkungsbetrag</label>
         <name>Matching_Gift_Amount_Min__c</name>
     </fields>
     <fields>
-        <help>Der maximale Betrag, den das Unternehmen pro Jahr einem Mitarbeiter zuweist</help>
-        <label>Zugew. Jahresspende f. Mitarb. max.</label>
+        <help>Der maximale Betrag, den das Unternehmen pro Jahr mit einem Mitarbeiter abgleicht</help>
+        <label>Abgegl. Schenkung/Jahr f. Mitarb. max.</label>
```

Comment:
@force2b I want to update this translation just for German to be consistent with the rest of the changes for the term "Matching_Gift" - what's the best way to send it to you? 
New translation: Übereinst. Schenkung/Jahr f. Mitar. max.

Reply:
Is it just this one label or are there other changes in this file?

Reply:
Just this 1 label @force2b

Reply:
@GloryChoe I can always change this one label manually. We can do the same for the missing SuccessMessage translation on the QuickAction that I noted earlier. The key for these is that these two manually updated translations are not lost/reverted when we do the next full translation in a few months.

Reply:
@force2b Thanks Michael. Yes please enter in the new German translation. It won't get overwritten next time because I ensured we have it fixed on our loc repo and our translation tool.

As for the successMessage, I will get the translation delivered for you to enter in manually. I also have let Michael Fang take a look into potentially supporting the loc for successMessage in the future.

Reply:
Thanks for the updates. I changed this translation and will apply the SuccessMessage translations when they come in next week.